index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
Ask questions to verify shared understanding.
is this a supporting text to an icon 
Highlight what the author did well and should keep doing.
this text encourages people to keep playing.